docs/getting-started/delta-lake-format.md

```diff
@@ -0,0 +1,178 @@
+
+## Reading Delta Lake Format with Spark
+
+Checking our results gets more complicated now - we can't just open delta lake format in a text editor like we used to do for CSV files.
+We could now use SQL to query our results, that would be even better. 
+One option is to use a Spark session, i.e. by starting a spark-shell.
+But state-of-the-art is to use notebooks like Jupyter for this.
+One of the most advanced notebooks for Scala code we found is Polynote, see [polynote.org](https://polynote.org/).
+
+We will now start Polynote in a docker container, and an external Metastore (Derby database) in another container to share the catalog between our experiments and the notebook.
+To do so, run the following commands in the projects root directory:
+    
+    docker-compose build
+    mkdir -p data/_metastore
+    docker-compose up
+
+This might take multiple minutes.
+You should now be able to access Polynote at `localhost:8192`. 
+
+:::info Docker on Windows
+If you use Windows, please read our note on [Docker for Windows](docker-on-windows).
+
+
+If you encounter the following error on Windows:
+
+`ERROR [stage-1 4/5] RUN chmod +x install_spark.sh && ./install_spark.sh`
+
+that probably means that the file `install_spark.sh` got automatically converted to CRLF Line Endings on your machine.
+You can fix the Problem by converting the file to LF Line endings.
+:::
+
+But when you walk through the prepared notebook "SelectingData", you won't see any tables and data yet. 
+Can you guess why?  
+This is because your last pipeline run used an internal metastore, and not the external metastore we started with docker-compose yet.
+To configure Spark to use our external metastore, add the following spark properties to the application.conf under global.spark-options. 
+You probably don't have a global section in your application.conf yet, so here is the full block you need to add at the top of the file:
+
+    global {
+      spark-options {
+        "spark.hadoop.javax.jdo.option.ConnectionURL" = "jdbc:derby://metastore:1527/db;create=true"
+        "spark.hadoop.javax.jdo.option.ConnectionDriverName" = "org.apache.derby.jdbc.ClientDriver"
+        "spark.hadoop.javax.jdo.option.ConnectionUserName" = "sa"
+        "spark.hadoop.javax.jdo.option.ConnectionPassword" = "1234"
+      }
+    }
+
+This instructs Spark to use the external metastore you started with docker-compose. 
+Your Smart Data Lake container doesn't have access to the other containers just yet. 
+So when you run your data pipeline again, you need to add a parameter `--network getting-started_default` to join the virtual network where the metastore is located:
+
+    docker run --rm -v ${PWD}/data:/mnt/data -v ${PWD}/config:/mnt/config --network getting-started_default smart-data-lake/gs1:latest -c /mnt/config --feed-sel '.*'
+
+When using podman you need to join the pod where the metastore is located with `--pod getting-started`:
```

Comment:
I installed podman. When I reacht this step, I have no pod after running podman compose up. Should I create a pod manually or should it have been created by the command podman compose up?

Reply:
it should be created automatically. Check with "podman pod ls".